### PR TITLE
docs: add task for wasm extensions

### PR DESCRIPTION
**Pull request by zhaohuabing**

### CODE REVIEW

File: site/content/en/latest/tasks/extensibility/wasm.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+* Image Wasm Extension: The Wasm extension is packaged as an OCI image and fetched from an image registry.
+
+The following example demonstrates how to configure an [EnvoyExtensionPolicy][] to attach a Wasm extension to an [EnvoyExtensionPolicy][] .
+This Wasm extension adds a custom header `x-wasm-custom: FOO` to the response.
+
+### HTTP Wasm Extension
+
+This [EnvoyExtensionPolicy][] configuration fetches the Wasm extension from an HTTP URL.
+
+```yaml
```
Comment (shawnh2) [inline site/content/en/latest/tasks/extensibility/wasm.md:37]: format error ?

File: site/content/en/latest/tasks/extensibility/wasm.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+
+```shell
+kubectl get envoyextensionpolicy/http-wasm-source-test -o yaml
+```
+
+### Image Wasm Extension
+
+This [EnvoyExtensionPolicy][] configuration fetches the Wasm extension from an OCI image.
+
+```yaml
```
Comment (shawnh2) [inline site/content/en/latest/tasks/extensibility/wasm.md:102]: ditto

File: site/content/en/latest/tasks/extensibility/wasm.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+    group: gateway.networking.k8s.io
+    kind: HTTPRoute
+    name: backend
+  wasm:
+  - name: wasm-filter
+    rootID: my_root_id
+    code:
+      type: Image
+      image:
+        url: zhaohuabing/testwasm:v0.0.1
```
Comment (Xunzhuo) [inline site/content/en/latest/tasks/extensibility/wasm.md:120]: Should add a few lines about what this image workes for: like add a header or what

Comment (zhaohuabing) [reply site/content/en/latest/tasks/extensibility/wasm.md:120]: Line 30:

The following example demonstrates how to configure an [EnvoyExtensionPolicy][] to attach a Wasm extension to an [EnvoyExtensionPolicy][] .
This Wasm extension adds a custom header `x-wasm-custom: FOO` to the response.

**Review by shawnh2 — APPROVED** [review]
lgtm

**Review by zirain — APPROVED** [review]

**Pull request merged**